project/Dependencies.scala

```diff
@@ -516,16 +516,16 @@ object Dependencies {
 
   val bcsBackendDependencies: List[ModuleID] = commonDependencies ++ refinedTypeDependenciesList ++ aliyunBatchComputeDependencies
 
-  val tesBackendAzureDependencies: List[ModuleID] = List(
+  val azureDependencies: List[ModuleID] = List(
```

Comment:
I thought the reason for having "tes" in the name is that some of these dependencies are needed for the TES backend. Need to go back through git history.

Reply:
Previously these dependencies were used exclusively in the TES backend, now they're being used in the standalone DRS localizer as well.

Reply:
But in just the past couple of days, this has gone from being called `tesBackendDependencies` to `azureDependencies` and I'm afraid we're losing context with that change. Maybe we should split these out into a couple sublists and join them together as needed depending on the artifact being built.

Reply:
Ah - during review in my recent PR that added this, there was a desire to handle these in a separate list and call them out as Azure-specific and not general TES dependencies. Note that this list we're talking about wasn't ever called `tesBackendDependencies`, rather these libraries used to be part of the existing `tesBackendDependencies` list, rather than a separate list that is included in `tesBackendDependencies`.

I don't know if that clarifies anything or further confuses things, happy to chat about this off-github if that's easier. I do think that the current state is a good representation of how the libraries are being used.

Reply:
Yeah... it would help if I read the code correctly. 🙄  Thank you for clarifying.